.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: build
+on: [push, pull_request]
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
```

Comment:
Is this still the right Go version? It looks like they [released 1.16 in 2021-02](https://golang.org/project).

Reply:
Thanks! Fixed.
It seems we have to specify a specific version.
We can't specify "latest" or similar general term for Go.

I referred to https://github.com/actions/go-versions/blob/main/versions-manifest.json